index.d.ts

```diff
@@ -148,7 +148,7 @@ interface PlayerMp extends EntityMp {
 	readonly voiceListeners: PlayerMp[];
 
 	ban(reason: string): void;
-	call(eventName: string, args: any[]): void;
+	call(eventName: string, args?: any[]): void;
```

Comment:
Maybe just change it to `call(eventName: string, ...args: any[]): void`

Reply:
Hi, Vincent!
Seems that the same change was reverted a while ago.
https://github.com/CocaColaBear/types-ragemp-s/pull/14/files
Also, I have checked documentation of rage:mp.
You should pass arguments as an array, not as multiple arguments.
https://wiki.rage.mp/index.php?title=Player::call

Reply:
Alright.